app/views/bookmarks/index.html.erb

```diff
@@ -0,0 +1,17 @@
+
+<ul>
+  <% @bookmarks.each do |bookmark| %>
+  <!--  "== true" is not needed -->
+    <li>
+      <% if bookmark.favorite == true %> * <% end %>
```

Comment:
if `bookmark.favorite` returns a boolean value, you can leave out the `== true` part